testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/management/cli/DataSourceTestCase.java

```diff
@@ -114,6 +114,7 @@ private void testRemoveDataSource() throws Exception {
         // remove data source
         //cli.sendLine("data-source remove --name=TestDS");
         cli.sendLine("/subsystem=datasources/data-source=TestDS:remove{allow-resource-service-restart=true}");
+        cli.sendLine("/:reload");
```

Comment:
This should be:

cli.sendLine("reload");

When you use the high level CLI reload command instead of the low level ":reload" op, the CLI knows to expect the connection to the server to close, handles that cleanly, and reconnects. Any low level op like (i.e. the :xxx syntax) it just parses and sends a DMR op and then expects a response.